File: Assignment/Linked_List.java
package linked_list_ds;
class node
{
    int data;
    node next;
    node(int a)
    {
        data=a;
        next=null;
    }
    node head,first,second,temp;
   
}
public class Linked_List_DS {
    public static void main(String[] args) {
        node head = new node(10);
        node first = new node(30);
        node second=new node(50);
        head.next=first;
        first.next=second;
        second.next=null;
        int display()
        {
            while(head!=null)
            {
                System.out.print(head.data+" ");
                head=head.next;
            }
        }
        
    }
    
}

File: Assignment/Linked_List.c
#include <stdio.h>
#include <stdlib.h>
struct nd
{
    int data;
    struct nd *next;
};
typedef struct nd node;
node *head,*first,*second;
int fun(int a,int b,int c)
{
    head=(node*)malloc(sizeof(node));
    first=(node*)malloc(sizeof(node));
    second=(node*)malloc(sizeof(node));
    head->data=a;
    head->next=second;
    first->data=b;
    first->next=NULL;
    second->data=c;
    second->next=first;
}
int main() {
    fun(10,20,30);
    while(head!=NULL)
    {
        printf("%d ",head->data);
        head=head->next;
    }
}

